.gitignore

```diff
@@ -31,4 +31,4 @@ erl_crash.dump
 /deps_docker_compose
 
 # seeds
-*seeds.exs
+/priv/repo/seeds.exs
```

Comment:
Why ignore this now?  (I had assumed you wanted these dev seeds checked in).

Reply:
no particular reason! hadn't realized i'd checked them in before. but they don't hurt. will remove this!